packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -32,6 +34,8 @@ import '../../exported';
 import '../internal-types';
 
 export function buildRootObject(powers, _params, testJigSetter = undefined) {
+  console.log('makeKind in zcf', makeKind());
+  console.log('makeWeakStore in zcf', makeWeakStore());
```

Comment:
are these `console.log` statements temporary? I usually mark my temporary stuff with '@@'.

or is this advice out of date?
> In general we need to remove unsolicited console.logs from our codebase
-- https://github.com/Agoric/agoric-sdk/wiki/agoric-sdk-unit-testing#patterns

Reply:
Temporary for debugging purposes. This is a draft